api/src/utils/fme-import-utils.js

```diff
@@ -21,7 +21,7 @@ const NRIS_FLNRO_KEY = 'fme-exports/NRIS-FLNRO-Inspections.csv';
  * @param {string} recordType record type to target, Eg. Inspection
  * @returns {string[]}
  */
-exports.getAndParseFmeCsv = async function(dataSourceType, recordType) {
+exports.getAndParseFmeCsv = function(dataSourceType, recordType) {
```

Comment:
This is an incorrect adjustment to the function.  Inside there is  `readAndParseCsvFile` which is defined as an async function in the utility file.  It is awaited in side the utility, therefore all callers must await as well.

Reply:
Ah yes, I missed that the `readAndParseCsvFile` should have await on it in my review